app/models/wakes/resource.rb

```diff
@@ -22,9 +22,10 @@ class Wakes::Resource < ActiveRecord::Base
   validates :label, :presence => true
 
   def to_s
+    legacy_location_labels = legacy_locations.pluck(:host, :path).map { |x| x.join('') }
```

Comment:
@benhutton I saw this pattern in the specs, and thought it was followed to avoid instantiating all of the legacy `Wakes::Location`s (since `pluck` retrieves _only_ `host` and `path` from the database) for debug code. It appears this happens anyway as part of a `Wakes.redirect`, though, so I'll revert back to the instance methods.